tests/saitests/sai_qos_tests.py

```diff
@@ -1755,23 +1755,29 @@ def runTest(self):
 
         # send packets
         try:
+            # Read the initial watermark as baseline
+            pg_shared_wm_res_bases = sai_thrift_read_pg_shared_watermark(self.client, port_list[src_port_id])
```

Comment:
There is no need to read baseline. before we run this test, we call the resetWatermark fixture which is to ensure that the watermark counters are cleared. we send 1 packet in get_rx_port. that will not affect the shared watermark counters.

Reply:
Updated the logic. Please check.